### ISSUE TITLE
Issue using Circuitscape

### ISSUE THREAD

**Issue by linhunt** [Circuitscape/Omniscape.jl#120]
This is the error I am getting when trying to run Circuitscape today. It told me to submit a bug report.

Step 10: Would you like to run Circuitscape?
 > Yes
   Later
[ Info: 2022-06-27 09:07:09 : Precision used: Double
[ Info: 2022-06-27 09:07:09 : Starting up Circuitscape to use 4 processes in parallel
[ Info: 2022-06-27 09:07:21 : Reading maps
[ Info: 2022-06-27 09:07:26 : Resistance/Conductance map has 118235658 nodes
┌ Warning: 2022-06-27 09:07:32 : ncols is not the same
└ @ Circuitscape C:\Users\lhunt\.julia\packages\Circuitscape\9x9VD\src\logging.jl:25
[ Info: 2022-06-27 09:07:45 : Total number of pair solves = 31375
[ Info: 2022-06-27 09:07:45 : Solving pair 1 of 31375
[ Info: 2022-06-27 09:10:11 : Solver used: CHOLMOD
[ Info: 2022-06-27 09:10:11 : Graph has 118235394 nodes, 2 focal points and 1 connected components
ERROR: ArgumentError: factorization construction failed for unknown reasons. Please submit a bug report.
Stacktrace:
 [1] SuiteSparse.CHOLMOD.Factor{Float64}(::Ptr{SuiteSparse.CHOLMOD.C_Factor{Float64}}, ::Bool) at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\SuiteSparse\src\cholmod.jl:338
 [2] Factor at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\SuiteSparse\src\cholmod.jl:337 [inlined]
 [3] Factor at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\SuiteSparse\src\cholmod.jl:359 [inlined]
 [4] analyze(::SuiteSparse.CHOLMOD.Sparse{Float64}, ::Array{UInt8,1}) at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\SuiteSparse\src\cholmod.jl:719
 [5] #fact_#5 at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\SuiteSparse\src\cholmod.jl:1289 [inlined]
 [6] cholesky(::SuiteSparse.CHOLMOD.Sparse{Float64}; shift::Float64, check::Bool, perm::Nothing) at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\SuiteSparse\src\cholmod.jl:1343
 [7] cholesky at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\SuiteSparse\src\cholmod.jl:1339 [inlined]
 [8] #cholesky#9 at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\SuiteSparse\src\cholmod.jl:1458 [inlined]
 [9] cholesky at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\SuiteSparse\src\cholmod.jl:1458 [inlined]
 [10] construct_cholesky_factor at C:\Users\lhunt\.julia\packages\Circuitscape\9x9VD\src\core.jl:464 [inlined]
 [11] macro expansion at .\timing.jl:233 [inlined]
 [12] _cholmod_solver_path(::Circuitscape.GraphData{Float64,Int64}, ::Circuitscape.RasterFlags, ::Dict{String,String}, ::Bool, ::Int64) at C:\Users\lhunt\.julia\packages\Circuitscape\9x9VD\src\core.jl:332
 [13] single_ground_all_pairs(::Circuitscape.GraphData{Float64,Int64}, ::Circuitscape.RasterFlags, ::Dict{String,String}, ::Bool) at C:\Users\lhunt\.julia\packages\Circuitscape\9x9VD\src\core.jl:59
 [14] _pt_file_polygons_path(::Circuitscape.RasData{Float64,Int64}, ::Circuitscape.RasterFlags, ::Dict{String,String}) at C:\Users\lhunt\.julia\packages\Circuitscape\9x9VD\src\raster\pairwise.jl:105
 [15] raster_pairwise(::Type{T} where T, ::Type{T} where T, ::Dict{String,String}) at C:\Users\lhunt\.julia\packages\Circuitscape\9x9VD\src\raster\pairwise.jl:27
 [16] _compute(::Type{T} where T, ::Type{T} where T, ::Dict{String,String}) at C:\Users\lhunt\.julia\packages\Circuitscape\9x9VD\src\run.jl:42
 [17] macro expansion at .\timing.jl:233 [inlined]
 [18] compute(::Dict{String,String}) at C:\Users\lhunt\.julia\packages\Circuitscape\9x9VD\src\run.jl:73
 [19] step10(::String, ::String) at C:\Users\lhunt\.julia\packages\Circuitscape\9x9VD\src\INIBuilder\run.jl:139
 [20] step9(::String) at C:\Users\lhunt\.julia\packages\Circuitscape\9x9VD\src\INIBuilder\run.jl:131
 [21] step8() at C:\Users\lhunt\.julia\packages\Circuitscape\9x9VD\src\INIBuilder\run.jl:120

**Comment by ranjanan** [comment]
Think you put this in the wrong repo. I'll file an issue at CS.jl. Could you please the files there so I can reproduce?

**Comment by ranjanan** [comment]
Generate the INI and paste it there with the raster and point file

**Comment by linhunt** [comment]
THis isn't complete it wouldn't let me submit the comments

**Comment by vlandau** [comment]
Sorry for the delay in response here. This is indeed in the wrong repo. I'm going to close this issue since it is related to Circuitscape instead of Omniscape. It looks like @ranjanan has already created a new issue over in Circuitscape so please follow up on that issue.
